### ISSUE TITLE
Travis failures caused by cppcheck

### ISSUE THREAD

**Issue by smlng** [rtrlib/rtrlib#61]
Travis fails for some branches and PRs though the respective code compiles without error, ```cppcheck``` is raising several warnings/errors. It does not fail in master branch, because Travis uses an older version of cppcheck there. However, by applying #58 Travis will use a newer version of Ubuntu and cppcheck.

I would suggest to disable cppcheck in Travis and use it as an offline tool for developers. 

Btw. the build is not aborted automatically, just because ```cppcheck``` fails.

**Comment by waehlisch** [comment]
Why disabling if it now works under travis?

**Comment by smlng** [comment]
As said above: cppcheck on master currently works because it uses standard Travis container with an older cppcheck version. With PR #58 rtrlib will use newer Travis container with newer cppcheck version, with the latter raising more issues resulting in a build fail -> but only for cppcheck, the code compiles. 

From what I saw in Travis output for my PR #58 there quite a number of issue in the code according to cppcheck - and I honestly don't know if they can be fixed quickly, @fho is looking into it. 

In summary: PR #58 fails in Travis but only for cppcheck, code compiles! Thus, merging the PR will cause Travis to fail for master in the future. Unless we fix all issues raised by cppcheck or disable the check in Travis config (for the time being).

**Comment by waehlisch** [comment]
@smlng got it! Then let's wait for @fho.

**Comment by smlng** [comment]
Is fixed, closing!
